infra/elasticsearch/variables.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_version = "~> 0.11.0"
+
+  backend "s3" {
```

Comment:
Stupid question: Any reason why we're keeping this in a separate state file, separate from the rest of terraform stuff? 

Also, don't know if these files should go under `infra/terraform` (they're terraform files after all). But maybe you put them here to avoid confusion by mixing with the rest?

Reply:
I don't consider this part of the cluster in that it's purely a store for a backup in order to perform an upgrade.  I just also like separation rather than the terralith model.  If it were a scheduled backup, I'd consider it a cluster operation.